cluster-autoscaler/provisioningrequest/checkcapacity/provisioningclass.go

```diff
@@ -139,7 +143,11 @@ func (o *checkCapacityProvClass) checkcapacity(unschedulablePods []*apiv1.Pod, p
 	err, cleanupErr := clustersnapshot.WithForkedSnapshot(o.context.ClusterSnapshot, func() (bool, error) {
 		st, _, err := o.schedulingSimulator.TrySchedulePods(o.context.ClusterSnapshot, unschedulablePods, scheduling.ScheduleAnywhere, true)
 		if len(st) < len(unschedulablePods) || err != nil {
-			conditions.AddOrUpdateCondition(provReq, v1.Provisioned, metav1.ConditionFalse, conditions.CapacityIsNotFoundReason, "Capacity is not found, CA will try to find it later.", metav1.Now())
+			if _, ok := provReq.Spec.Parameters[NoRetryParameter]; ok {
```

Comment:
Do we really want to assume any value for "noRetry" means "true"?

Even if user inputs "false"?

Reply:
Fixed